### PR TITLE
fixes #1890 api host status

### PR DESCRIPTION
**Pull request by isratrade**
I took at same status names as the dashboard UI hover hints
-  "Pending Installation"
-  "Alerts disabled"
-  "No reports"
-  "Out of sync"
-  "Error"
-  "Active"
-  "Pending"
-  "No changes"

### CODE REVIEW

**Comment by isratrade** [comment]
@ohadlevy, generating the examples.yml doesn't give the responses in the same order, so it shows a bunch of additions/deletions, but only the hosts/:id/status example should have changed.

File: app/controllers/api/v1/hosts_controller.rb [hunk 1]
```diff
@@ -82,6 +82,38 @@ def destroy
...
+* pending
+* changed
+* unchanged
+* unreported
+
+     eos
+
+      def status
+        if @host
+          if Host.my_hosts.search_for('not has last_report and status.enabled = true').where(:id => @host.id).count > 0
```
Comment (ohadlevy) [inline app/controllers/api/v1/hosts_controller.rb]: Why not use the host status directly?

**Comment by isratrade** [comment]
@ohadlevy, I updated the status names from the "radiator" names to the dashboard UI hover hints

**Comment by ohadlevy** [comment]
rebased, clean up whitespaces and merged - thanks!

**Pull request closed without merging**